plugins/inputs/redfish/redfish.go

```diff
@@ -325,18 +367,20 @@ func (r *Redfish) Gather(acc telegraf.Accumulator) error {
 		}
 
 		for _, j := range power.PowerControl {
-			tags := map[string]string{
-				"member_id": j.MemberID,
-				"address":   address,
-				"name":      j.Name,
-				"source":    system.Hostname,
-			}
-			if chassis.Location != nil {
+			tags := map[string]string{}
+			tags["member_id"] = j.MemberID
+			tags["address"] = address
+			tags["name"] = j.Name
+			tags["source"] = system.Hostname
```

Comment:
Please leave the initialization as-is. This is nicer to read and allows the compiler to allocate it correctly.